core/templates/dev/head/base.html

```diff
@@ -225,9 +225,35 @@
                             </ul>
                           </li>
                         {% endif %}
-                      </ul>
-                    </div>
-                  {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if="windowWidth > 700"></create-exploration-button>
+                          <li ng-if="windowWidth > 700" class="dropdown oppia-clickable-dropdown pull-right" >
+                            <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" href="/forum">
+                              Forum
+                            </a>
+                          </li>
+                          <li ng-if="windowWidth > 700" class="dropdown oppia-clickable-dropdown pull-right">
+                            <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" >
+                              About
+                            </a>
+                            <ul ng-if="windowWidth > 700" class="dropdown-menu oppia-navbar-profile-dropdown" ng-mouseover="onMouseoverDropdownMenu($event)" ng-mouseleave="onMouseoutDropdownMenu($event)">
+                              <li><a href="/about">About Oppia</a></li>
+                              <li><a href="/participate">Participation Playbook</a></li>
+                              <li><a href="http://oppiablog.blogspot.com/">Blog</a></li>
+                              <li><a href="/terms">Terms of Service</a></li>
+                              <li><a href="/privacy">Privacy Policy</a></li>
+                            </ul>
+                          </li>
+                          <li ng-if="windowWidth > 700" class="dropdown oppia-clickable-dropdown pull-right">
+                            <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" href="/gallery">
+                              Explore
+                            </a>
+                          </li>
+                        {% endif %}
+                </div>
```

Comment:
No, I mean that the `</div>` should be two spaces deindented from the line before it, and similarly for the next line. The nested structure should be clear, and the indentation level of each "opening bracket" should match that of the corresponding "closing bracket".